sql-cli/sql_cli/__main__.py

```diff
@@ -294,6 +294,31 @@ def init(
     rprint("Initialized an Astro SQL project at", project.directory)
 
 
+@app.command(
+    cls=AstroCommand,
+    help="""Deploy workflows to Astro Cloud.""",
+)
+def deploy(
+    workflow_name: str = typer.Option(  # skipcq: PYL-W0613
+        default=None,
+        show_default=False,
+        help="name of the workflow directory within workflows directory.",
+    ),
+    env: str = typer.Option(  # skipcq: PYL-W0613
+        metavar="environment",
+        default="default",
+        help="environment to deploy",
+    ),
+    project_dir: Path = typer.Option(  # skipcq: PYL-W0613
```

Comment:
Should we also add the argument `-y`?
```
There should be an optional flag -y which, if passed, will give empower the application to make decisions on behalf of the user, without prompting.
```
https://github.com/astronomer/astro-sdk/issues/1520

Reply:
I don't think we need that. All the flags/options in the cobra flow deploy command can be set in the command. If so, it won't prompt.

Reply:
we discussed about this on a call and both @feluelle @pankajkoti and I agreed the flag is valuable, but we'll address it in a dedicated ticket - so we don't put the 0.3 release in risk

Reply:
https://github.com/astronomer/astro-sdk/issues/1618